kuma/wiki/constants.py

```diff
@@ -661,3 +661,6 @@
 WIKI_ONLY_DOCUMENT_QUERY_PARAMS = frozenset(
     ("edit_links", "include", "macros", "nomacros", "raw", "section", "summary",)
 )
+
+
+NOINDEX_SLUG_STARTS = ("MDN/Doc_status",)
```

Comment:
Nit: is `SLUG_STARTS` a good name here, or would `SLUG_PREFIXES` be more clear?

Possible premature optimization: Are we sure that we'll only be concerned with prefixes, or might we be better served by a list of patterns instead, say, globs or regexps?

Reply:
Yeah, "PREFIXES" might be more accurate. 
I can't imagine there being a need for something that isn't just prefixes. If I'm wrong, we'll deal with it the day it matters. Ok?